src/test/ts/EqTest.ts

```diff
@@ -100,37 +101,37 @@ describe('eqAny', () => {
     assert.strictEqual(eqAny.eq(undefined, ''), false);
     assert.strictEqual(eqAny.eq(undefined, []), false);
     assert.strictEqual(eqAny.eq(undefined, {}), false);
-    assert.strictEqual(eqAny.eq(undefined, {a: NaN}), false);
+    assert.strictEqual(eqAny.eq(undefined, { a: NaN }), false);
     assert.strictEqual(eqAny.eq(undefined, () => 3), false);
 
     assert.strictEqual(eqAny.eq([], []), true);
-    assert.strictEqual(eqAny.eq([0], [0]), true);
-    assert.strictEqual(eqAny.eq([[0]], [[0]]), true);
-    assert.strictEqual(eqAny.eq([[[0]]], [[[0]]]), true);
-    assert.strictEqual(eqAny.eq(['cat', {x: 3}, 0, null, undefined], ['cat', {x: 3}, 0, null, undefined]), true);
-    assert.strictEqual(eqAny.eq(['cat', {x: 3, y: 8}, 0, null, undefined], ['cat', {x: 3}, 0, null, undefined]), false);
+    assert.strictEqual(eqAny.eq([ 0 ], [ 0 ]), true);
+    assert.strictEqual(eqAny.eq([[ 0 ]], [[ 0 ]]), true);
+    assert.strictEqual(eqAny.eq([[[ 0 ]]], [[[ 0 ]]]), true);
+    assert.strictEqual(eqAny.eq([ 'cat', { x: 3 }, 0, null, undefined ], [ 'cat', { x: 3 }, 0, null, undefined ]), true);
+    assert.strictEqual(eqAny.eq([ 'cat', { x: 3, y: 8 }, 0, null, undefined ], [ 'cat', { x: 3 }, 0, null, undefined ]), false);
   });
 
   it('eqs with different key order', () => {
     assert.strictEqual(eqAny.eq(
-      {a: 'cat', b: 'dog'},
-      {b: 'dog', a: 'cat'}
+      { a: 'cat', b: 'dog' },
+      { b: 'dog', a: 'cat' }
     ), true);
   });
 });
 
 describe('contramap', () => {
   it('contramaps', () => {
-    type Numbo = {
+    interface Numbo {
```

Comment:
Huh. Didn't even know you could make types inside a code block like this. Is that a local type then? Looks weird should we have a linter rule for that?

Reply:
Yeah you can declare them anywhere basically... I believe it'd only be scoped locally but I'm not sure. We generally do not allow this though and always prefer them to be at the top of the file after imports. I'll fix this up as this was an auto fix and I missed it.